sqlrunner/main.py

```diff
@@ -35,7 +37,13 @@ def main(args):
 
 
 def generate_column_headers(dummy_data_file, sql_query):
-    raise NotImplementedError
+    tsql_parser = TSQL()
+    parsed = [q for q in tsql_parser.parse(sql_query) if q]
+    # expand out "*" to full column names
+    columns = qualify_columns(parsed[0], schema=None, infer_schema=True).named_selects
```

Comment:
A comment or two about what's going on here would be useful, especially as 5d6d410 suggests that it isn't working as intended.